### PR TITLE
add v2 migration doc

### PR DESCRIPTION
**Pull request by assimovt**

### CODE REVIEW

File: migration/v2.mdx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+description: Everything you need to know about migrating from sync v1 to v2.
+---
+
+## Overview
+
+We are excited to introduce **sync.v2**, a major overhaul of our platform that includes a redesigned playground and the release of the new lipsync model, `lipsync-1.8.0-beta`.
+
+sync. v2 brings significant improvements to the user experience, performance, and the underlying technology.
+
+If you're reading this, your account has already been migrated from sync v1 to v2.
```
Comment (assimovt) [inline migration/v2.mdx]: think we can drop this, as we have in the email

File: migration/v2.mdx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+- **Redesigned Playground**: Enjoy an improved, more intuitive playground that allows you to compare models, fine-tune generation options, and quickly preview API requests, helping you easily replicate the same functionality in your integrations.
+- **New `lipsync-1.8.0-beta` model**: Experience enhanced lip synchronization with major improvements to teeth and mouth generation.
+- **Unified API**: Streamlined integration through a single, unified API endpoint, simplifying your workflow and future-proofing your applications.
+
+## Migration Steps
+
+### **Access the new Playground**
+
+- **Where to log in**: Navigate to the new playground at [https://platform.sync.so](https://platform.sync.so).
+- **Automatic migration**: Your account has already been migrated to sync. v2, so simply log in with your existing credentials.
```
Comment (assimovt) [inline migration/v2.mdx]: can mention, if not wait for an email as we're doing in cohorts

File: migration/v2.mdx [hunk 3]
```diff
@@ -0,0 +1,59 @@
+---
+title: migrating from sync v1 to v2
+description: Everything you need to know about migrating from sync v1 to v2.
+---
+
+## Overview
+
+We are excited to introduce **sync.v2**, a major overhaul of our platform that includes a redesigned playground and the release of the new lipsync model, `lipsync-1.8.0-beta`.
+
+sync. v2 brings significant improvements to the user experience, performance, and the underlying technology.
```
Comment (satya-nutella) [inline migration/v2.mdx]: [nit] there seems to be a space here

**Review by rogue-yogi — APPROVED** [review]

**Review by nozma-knows — APPROVED** [review]
🚀

**Pull request merged**